content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -143,27 +151,44 @@ is recommended to use
 [taints on nodes](/docs/concepts/configuration/taint-and-toleration/)
 to schedule those pods onto the right nodes.
 
-## PodSecurityPolicy Annotations
+## PodSecurityPolicy
 
-The use of sysctl in pods can be controlled via annotation on the PodSecurityPolicy.
+The use of sysctl in pods can be controlled through `forbiddenSysctls` and
+`allowedUnsafeSysctls` fields on the PodSecurityPolicy.
 
-Sysctl annotation represents a whitelist of allowed safe and unsafe sysctls
-in a pod spec. It's a comma-separated list of plain sysctl names or sysctl patterns
-(which end in `*`). The string `*` matches all sysctls.
+By default, all safe sysctls are allowed. Currently, the whitelist of safe
+sysctls corresponds to the safe set mentioned above.
 
-Here is an example, it authorizes binding user creating pod with corresponding sysctls.
+Both `forbiddenSysctls` and `allowedUnsafeSysctls` are lists of plain sysctl names
+or sysctl patterns (which end with `*`). The string `*` matches all sysctls.
+
+The `forbiddenSysctls` field excludes sysctls from both safe and unsafe set (`*` means
+no sysctl allowed).
+Any unsafe sysctl specified in the `allowedUnsafeSysctls` (and not in the `forbiddenSysctls`)
+is allowed (`*` means all unsafe sysctls allowed).
+In case a plain sysctl (or sysctl pattern) overlap in both lists, the PSP is rejected.
```

Comment:
nit: overlaps

Reply:
"the PSP is rejected" sounds strange. Better write `forbiddenSysctls` and `allowedUnsafeSysctls` must not overlap.